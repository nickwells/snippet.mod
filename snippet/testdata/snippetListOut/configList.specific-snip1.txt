    /home/ngw/go/src/github.com/nickwells/snippet.mod/snippet/testdata/testListConfig/snip1
           Note: snip1 - Note
