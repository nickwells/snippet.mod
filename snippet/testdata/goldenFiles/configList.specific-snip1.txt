in: testdata/testListConfig

    snip1
           Note: snip1 - Note
